roles/etcd/meta/main.yml

```diff
@@ -7,5 +7,4 @@ dependencies:
     file: "{{ downloads.etcd }}"
     when: etcd_deployment_type == "host"
   - role: docker
-    when: (ansible_os_family != "CoreOS" and etcd_deployment_type == "docker")
-  - role: "kubernetes/preinstall"
+    when: (ansible_os_family != "CoreOS" and etcd_deployment_type == "docker" or inventory_hostname in groups['k8s-cluster']")
```

Comment:
We still have host mode etcd. Why not attach docker to kubernetes/preinstall instead?